apps/studio/src/lib/db/authentication/azure.ts

```diff
@@ -153,17 +171,64 @@ export class AzureAuthService {
     }
   }
 
+  private async getAzureCLIToken(options: AzureAuthOptions): Promise<AuthConfig> {
+
+    if(!options?.cliPath){
+      throw new Error('AZ command not specified')
+    }
+
+
+    return new Promise<AuthConfig>((resolve, reject) => {
+      const proc = spawn(options?.cliPath, [
+      'account',
+      'get-access-token',
+      '--resource',
+      BksConfig.azure.azSQLLoginScope,
+      '--output',
+      'json'
+    ]);
+
+      proc.stdout.on('data', (chunk) => {
+        if (chunk) {
+          const data: string = chunk.toString().trim();
+          const tokenData = JSON.parse(data);
```

Comment:
You have to collect the output, you can't parse it here, what if there are more chunks?